c2corg_ui/static/js/alertservice.js

```diff
@@ -41,10 +51,45 @@ app.Alerts.prototype.get = function() {
 
 
 /**
+ * @param {Object} response Response from the API server.
+ * @return {string}
+ * @private
+ */
+app.Alerts.prototype.formatErrorMsg_ = function(response) {
+  var errors = response['data']['errors'],
```

Comment:
Perhaps I should add an ``assert`` here to make sure that ``response['data']['errors']`` is available.